test/unit/netbox_test.lua

```diff
@@ -247,7 +247,7 @@ function g.test_errors_netbox_wait_async()
     }, 'netbox_wait_async (eval request timed out)')
 
     local csw2 = h.fiber_csw()
-    t.assert_equals(csw1, csw2, 'Unnecessary yield')
```

Comment:
In case of an error (which I saw a lot) it makes the message more clear:

```
/opt/errors/test/unit/netbox_test.lua:250: Unnecessary yield
expected: 55, actual: 57

vs

expected: 57, actual: 55
```